third_party/openmetadata/prepare_OM_for_fybrik.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	logging "fybrik.io/fybrik/pkg/logging"
+	openapi_connector_core "fybrik.io/openmetadata-connector/pkg/openmetadata-connector-core"
+)
+
+func main() {
+	conf := map[interface{}]interface{}{
+		"openmetadata_endpoint": "http://localhost:8585/api",
+	}
+	logger := logging.LogInit(logging.CONNECTOR, "OpenMetadata Prepare for Fybrik")
+	openapi_connector_core.NewOpenMetadataAPIService(conf, &logger)
```

Comment:
can we separate the connection and tag creation from NewOpenMetadataAPI Service creation, and get the custom properties and tags from external files

Reply:
done.